html/index.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
```

Comment:
The file is there to provide easy access to the produced JSON files. I'll remove the footer.